### DISCUSSION TITLE
Laravel 8 Sail Redis not assuming the default cache driver

### DISCUSSION THREAD

**Discussion by adellinocasas** [laravel/framework#37191] [General]
Laravel 8 and the dev environment is Sail - including Redis. The .env file:

`
CACHE_DRIVER=redis
..
REDIS_HOST=redis
REDIS_PASSWORD=null
REDIS_PORT=6379`

Using the Redis facade functions, like

`Redis::set( 'bar', 'The hole of the moon' );`

using redis-cli is possible to find the value stored in cache

but using

`Cache::put('bar2', 'baz', 100);`
or even

`Cache::store('redis')->put('bar3', 'Awesome', 100); `

the values cannot be seen using the redis console.

Please note: 
`dd(env('CACHE_DRIVER'));`
shows "redis" .

**Comment by devcircus** [comment]
Are you taking into consideration the prefix for the key? I believe Laravel prefixes all cache keys when using redis for cache. Check the cache config file at the bottom I think. Should be similar to "app_name_cache_bar" instead of "bar".

**Comment by adellinocasas** [comment]
Yes, all keys from the app are prefix with "laravel_database_".
This is the print from terminal 
`127.0.0.1:6379> keys *
1) "laravel_database_POSTS.ALL.TITLE"
2) "laravel_database_POSTS.ID.112"
3) "laravel_database_POST.COUNT.112"
4) "laravel_database_post.id.111"
5) "laravel_database_post.id.112"`
